src/rust/ide/src/double_representation/module.rs

```diff
@@ -147,23 +148,23 @@ impl Id {
 #[serde(into="String")]
 #[serde(try_from="String")]
 pub struct QualifiedName {
-    /// The first segment in the full qualified name.
-    pub project_name : ReferentName,
+    /// The first segment in the full qualified name. May be a project name or a keyword like
```

Comment:
> May be a project name or a keyword like
 
 This is not clear and looks as unfinished.